README.md

```diff
@@ -51,6 +51,10 @@ Copy `config/config-example.js` into `config/config.js`, and edit as you please.
 
 Congratulations, you're done setting up Pokémon Showdown.
 
+After you will need to install the dependencies:
+
+    npm install
```

Comment:
No, the console tells you when you type "node pokemon-showdown" to start it if the their not installed. But no harm on having it on the file.